web/src/components/layouts/layout.tsx

```diff
@@ -376,11 +381,12 @@ export default function Layout(props: PropsWithChildren) {
                         <a
                           onClick={() => void item.onClick()}
                           className={cn(
-                            active ? "bg-gray-50" : "",
-                            "block cursor-pointer px-3 py-1 text-sm text-gray-900",
+                            active ? "bg-primary-foreground" : "",
+                            "flex cursor-pointer items-center justify-between px-2 py-0.5 text-sm leading-6 text-primary",
                           )}
                         >
                           {item.name}
+                          {item.name === "Theme" && <ThemeToggle />}
```

Comment:
just add as custom element outside of userNavigation.map, this way this ifelsehas a condition on the item being in the list, not clear what is happening here to someone reading this